contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -8,749 +8,24 @@ doc: |
   
   .. _ISO 18115-1:2023: https://www.iso.org/standard/74811.html
   .. _IUPAC Recommendations 2020: https://doi.org/10.1515/pac-2019-0404
-symbols:
-  doc: |
-    The symbols used in the schema to specify e.g. dimensions of arrays
-  n_transmission_function: |
-    Number of data points in the transmission function.
 type: group
-NXmpes(NXobject):
+NXmpes(NXphotoemission):
   (NXentry):
-    definition:
-      \@version:
-      enumeration: [NXmpes]
```

Comment:
This should stay here, no? An NXmpes file conforms to NXphotoemission, but not the other way around.

Reply:
Yes, indeed. This was by mistake. I added it back.